crates/db_views_actor/src/person_view.rs

```diff
@@ -48,21 +49,17 @@ fn post_to_person_sort_type(sort: SortType) -> PersonSortType {
 
 fn queries<'a>(
 ) -> Queries<impl ReadFn<'a, PersonView, PersonId>, impl ListFn<'a, PersonView, ListMode>> {
-  let creator_is_admin = exists(
-    local_user::table.filter(
-      person::id
-        .eq(local_user::person_id)
-        .and(local_user::admin.eq(true)),
-    ),
-  );
   let all_joins = move |query: person::BoxedQuery<'a, Pg>| {
     query
       .inner_join(person_aggregates::table)
+      .left_join(local_user::table)
       .filter(person::deleted.eq(false))
       .select((
         person::all_columns,
         person_aggregates::all_columns,
-        creator_is_admin,
+        sql::<Bool>("COALESCE(")
```

Comment:
Add this line to the top level of this file somewhere:

`sql_function! { fn coalesce(x: Nullable<Bool>, y: Timestamptz) -> Bool; }`

And then use `coalesce(local_user::admin.nullable(), false)`

Reply:
It's already defined at `lemmy_db_schema::utils::functions::coalesce`